.github/workflows/update_gbs_cache.yml

```diff
@@ -12,7 +12,15 @@ jobs:
   cache_gbs_build:
     strategy:
       matrix:
-        gbs_build_arch: [x86_64, i586, armv7l, aarch64]
+        include:
+          - gbs_build_arch: "x86_64"
+            gbs_build_option: "--define \"unit_test 1\""
+          - gbs_build_arch: "i586"
+            gbs_build_option: "--define \"unit_test 1\""
+          - gbs_build_arch: "armv7l"
+            gbs_build_option: "--define \"unit_test 0\""
```

Comment:
It's changed to daily build, how about running test on armv7l?
Aarch64 seems to have build and test problem.

Reply:
In the end, both armv7l and aarch64 should enable unit tests.